### ISSUE TITLE
Create new Pages Low-impact CRM for customers derived from a canonical CRM/CIS

### ISSUE THREAD

**Issue by pburkholder** [cloud-gov/product#1604]
In order to better guide customers and compliance folks on Pages customer responsibilities for Low-Impact sites, we should generate a CRM for Pages that is:
* only the Low controls
* stands on its own w/o referencing the PaaS

## Acceptance Criteria
* [ ] GIVEN a customer requesting a Pages CRM for Low-Impact site \
THEN we can update the canonical CRM \
AND share a sheet from that specific to the customer needs

--- 

## Security considerations

CRM is public-facing, we need to exercise usual caution about sharing internal implementation details that are not in scope

## Implementation sketch

Use Google Sheets features to query the canonical CRM, also to replace, say, `(ditto)` in the Pages column with the text from the PaaS column.

**Comment by pburkholder** [comment]
I've updated all the Pages controls in https://docs.google.com/spreadsheets/d/1hawBfPs6fApMUALOEIT52LZIYGVQgDHp/edit#gid=765527261 so the statements that  simply read 

> "(No additional considerations for Pages sites)" 

actually had the appropriate text.

I then created a separate sheet that's only the Pages low controls using a Google `=query` statement.

Downloading the .xlsx should yield a readable Excel doc with a "Pages Low CRM" sheet.
